src/managers/AdManager/AdManager.ts

```diff
@@ -21,82 +26,89 @@ class AdManager {
 
   private page: TPages | null;
   private productsIds: ReturnType<typeof getProductsIds> | [];
-  private websiteId: number;
 
   public injectAdnPixelScript = () => {
     const area = this.page ? this.mapPageToArea(this.page) : null;
 
     if (window.dlApi && dlApi.fetchNativeAd) {
-      dlApi.area = area || '';
-      dlApi.keyvalues = {
-        offer_ids: this.productsIds.toString(),
-        website_id: this.websiteId,
-      };
+      dlApi.area = area || 'ros';
+      dlApi.addKeyValue('offer_ids', this.productsIds.toString());
```

Comment:
Docelowo będzie trzeba to przenieść do funkcji `dlApi.fetchNativeAd` - do argumentu `options.opts.offer_ids`. W przeciwnym razie wszystkie inne reklamy RMN na tej samej stronie (np. Promo Banner) również będą dopasowane kontekstowo do podanych ofert, a nie zawsze tego chcemy.